ledger/src/blockstore_processor.rs

```diff
@@ -513,9 +533,6 @@ fn process_entries_with_callback(
                             batch,
                             transaction_indexes,
                         });
-                        // entry is scheduled to be processed, transactions in it can be used to
-                        // update prioritization fee cache asynchronously.
-                        prioritization_fee_cache.update(bank.clone(), transactions.iter());
```

Comment:
To clarify, there was also a bug here in that we only updated fees when batches didn't conflict with currently grouped batches, instead of always when executing batches (fixed as part of this PR)?

Reply:
If an entry has conflict with already grouped "batches", then `! first_lock_err.is_ok()`, the `loop` will execute and empty currently grouped batches, then try to lock for that entry again. The second time locking should work, so prioritization_fee_cache would be updated (by transactions in that entry). If the locking still fail, then it's be down the path of bailing.

Reply:
oh I see now. got it!